File: 프로그래머스/unrated/181944. 홀짝 구분하기/홀짝 구분하기.js
const readline = require('readline');
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

let input = [];

rl.on('line', function (line) {
    input = line.split(' ');
}).on('close', function () {
    n = Number(input[0]);
    
    if(1 <= n && n <= 1000){
        const divide = n % 2;
        let str = "is";
        
        divide ? str += " odd" : str += " even";
        
        console.log(`${n} ${str}`);
    }else {
        console.log("숫자의 범위는 1 <= n <= 1000 입니다.")
    }
    
});

File: 프로그래머스/unrated/181944. 홀짝 구분하기/홀짝 구분하기.java
import java.util.Scanner;

public class Solution {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        
        if(1 <= n && n <= 1000){
            int divide = n % 2;
            String str = "";
            
            if(divide == 0) {
                str = "is even";
            }else {
                str = "is odd";
            }
            
            System.out.printf("%d %s", n, str);
        }
    }
}